docs/en/userguide/01_User_manual/02_Using_authenticator_apps.md

```diff
@@ -0,0 +1,41 @@
+title: Using authenticator apps
+summary: How to acquire and configure an Authenticator app for your phone
+
+# Using authenticator apps
+
+### Installing an authenticator app
+
+Before you can setup MFA on your account you’ll need to install an authenticator
+app on your phone which supports Time-based One-Time Passwords (TOTP), such as:
+
+- [Authy](https://authy.com/download/)
+- [Microsoft Authenticator](https://www.microsoft.com/en-us/account/authenticator)
```

Comment:
Any reason why "Google Authenticator" isn't mentioned? This is probably the most popular one.

Reply:
I'm glad you asked! I left it out because it's easily the worst of the three. No search, no visual distinction between registered methods, nigh or impossible to migrate / sync between devices. It feels like a barebones MVP, and it's been that way for years.

Microsoft Authenticator isn't a massive step up, but I've heard it referenced as in use by government clients recently, and I determined it to be a better option as an alternative to Authy.

Ultimately, given Google Authenticator's prevalence, you could argue that not including it suggests our MFA solution _doesn't_ work with it - so I suppose we should add it.

Reply:
Playing devils advocate, but I much prefer it over Authy :-)

Reply:
I think it's worth including. The list is by no means exhaustive, but Google Authentictor will remain a popular option, despite it's short comings. Microsoft Authenticator is good, but not much different. I think the same argument you've employed (a lot of folks use it in e.g. Govt.) is also applicable for Google Authenticator (maybe not Govt. specifically, but common none the less).